links/GateLink/GateLink.sc

```diff
@@ -36,7 +36,7 @@ GateLink : Link {
     ar { | input, id = "" |
         var signal = input;
         signal = Compander.ar(signal, signal,
-            thresh:     \gate_thresh.kr(thresh).dbamp,  // amplitude trigger threshold [-1, 1]
+			thresh:     \gate_thresh.kr(thresh).dbamp,  // amplitude trigger threshold [-1, 1]
```

Comment:
NIT but looks like there's extra indentation here

Reply:
true, thanks. SuperCollider's indentations are somehow differently interpreted than in XCode..

Reply:
submitting a new one